rest_framework/exceptions.py

```diff
@@ -145,3 +145,7 @@ def __init__(self, wait=None, detail=None):
             self.detail += ' ' + force_text(
                 self.extra_detail % {'wait': self.wait}
             )
+
+
+class RESTFrameworkSettingHasUnexpectedClassWarning(Warning):
+    pass
```

Comment:
Let's use python standard exceptions or Django built-in exceptions for configuration errors and suchlike.
We'll reserve our exceptions for user facing API exceptions.